worlds/tunic/grass.py

```diff
@@ -0,0 +1,6442 @@
+    "Guardhouse 2 - Guard House 2 Upper Grass (3) (155.5, 0.0, -13.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (32) (156.5, 0.0, -14.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Bush (9) (153.0, 0.0, -11.0)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (4) (152.5, 0.0, -12.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (10) (151.5, 0.0, -12.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (13) (151.5, 0.0, -9.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Bush (8) (151.0, 0.0, -11.0)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (8) (150.5, 0.0, -12.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (15) (150.5, 0.0, -9.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (35) (156.5, 0.0, -15.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (29) (155.5, 0.0, -19.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (30) (155.5, 0.0, -18.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (31) (154.5, 0.0, -19.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (28) (154.5, 0.0, -18.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (33) (157.5, 0.0, -15.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (34) (157.5, 0.0, -14.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Bush (3) (157.0, 0.0, -13.0)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (37) (157.5, 0.0, -11.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (38) (157.5, 0.0, -10.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (39) (156.5, 0.0, -11.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Grass (36) (156.5, 0.0, -10.5)": TunicLocationData("Guard House 2 Upper", "Guard House 2 Upper"),
+    "Guardhouse 2 - Guard House 2 Upper Bush (2) (155.0, 0.0, -11.0)": TunicL
```

Comment:
i stg github,,,, just once in your life work as expected

Reply:
Man, github really hates long files like this, doesn't it?

Reply:
at least it's consistent! haha

Reply:
If only -- the line numbers in the suggestion are different than the ones in the comment

Reply:
regardless you understand what i'm suggesting right? just, don't run the loop that applies the rule when you know it will only ever return True

Reply:
if they were all ad hoc and inside thousands of lines i wouldn't have pushed for and update at all, but since it's just one if before a loop i think it's very reasonable to make that update

Reply:
I didn't see any noticeable difference in gen times between the two, but it's theoretically more efficient so I made the change